velox/exec/tests/HashJoinTest.cpp

```diff
@@ -6217,6 +6217,34 @@ DEBUG_ONLY_TEST_F(HashJoinTest, reclaimDuringWaitForProbe) {
   ASSERT_EQ(reclaimerStats_.numNonReclaimableAttempts, 1);
 }
 
+DEBUG_ONLY_TEST_F(HashJoinTest, reclaimDuringFinishHashBuild) {
```

Comment:
Instead of adding a new test, can you fix arbitrationTriggeredByEnsureJoinTableFit? Thanks!